TombEditor.WPF/ToolWindows/RoomOptions.xaml

```diff
@@ -70,7 +70,8 @@
 					<ComboBox
 						x:Name="comboFlipMap"
 						Grid.Row="1"
-						Grid.Column="1">
+						Grid.Column="1"
+                        Width="58">
```

Comment:
I made that change to RoomOptions.xaml because the space was too tight for the content

|                                                   before                                                  | after                                                                                                     |
|:---------------------------------------------------------------------------------------------------------:|-----------------------------------------------------------------------------------------------------------|
| ![image](https://github.com/MontyTRC89/Tomb-Editor/assets/116632612/a0690aa5-7a3d-44f4-a812-4a3605d3970d) | ![image](https://github.com/MontyTRC89/Tomb-Editor/assets/116632612/2117994d-d7c3-439c-9518-55477d185c67) |